### DISCUSSION TITLE
Long deploy scripts

### DISCUSSION THREAD

**Discussion by 7FigureSwagger** [ConsenSys-archive/truffle#5242] [General Q&A]
Is there a known issue with using long scripts, and/or many scripts during migrations? I am attempting to deploy to moonbase and keep getting polling errors, and i have to call the migrations again. Most of the time is successfully picks up where i left off, but it seems like kind of a waste of time. Wondering if others have had similar issues deploying to moonbeam and/or their testnet. 
My question is based on me assuming i'm doing something wrong here....

**Comment by apdan510** [comment]
Any chance you can share your repo? I would like to see more parameters, how long the script is, etc. 

Kind regards,
Alex | Truffle Support

**Reply by 7FigureSwagger** [reply]
Currently on a private gitlab instance, but let me copy it to my personal github and I will get you access.

**Reply by 7FigureSwagger** [reply]
Ok, you should have access now, you can see what i have going on in the migrations folder. Thanks!

**Reply by apdan510** [reply]
@7FigureSwagger 
Going to run some tests with this repo. I will let you know what I find. Thank you!
-Alex

**Reply by 7FigureSwagger** [reply]
> @7FigureSwagger Going to run some tests with this repo. I will let you know what I find. Thank you! -Alex

You're amanzing! Thanks brother 🤙🏽

**Reply by apdan510** [reply]
@7FigureSwagger I think I am looking at the wrong repo here. Is it the Dao-frontend repo?

**Reply by 7FigureSwagger** [reply]
> 

sorry for the delay, this is the repo!
https://github.com/7FigureSwagger/SparkNRG

EDIT: I just re-sent you the invite for the repo, it said it expired, i have this repo set to private i think thats why haha sorry!
